nestkernel/nest_time.h

```diff
@@ -473,6 +485,7 @@ class Time
   {
```

Comment:
In commit d05335a04bd81bbe56431641e8c49faa88c9d2fb it appears in the right place... but not in the PR discussion it seems. To reduce the confusion I created, I was referring to `calculate_steps()` in `Time(tic_t)` and `Time()`.  By caching there, under some usage patterns, you'll have a huge number of caching steps that are never used. For example, in `vector<Time>` or in temporaries created in computations.

But I believe downstream we've already agreed to just not do this, and do lazy evaluation in the Event class.